### DISCUSSION TITLE
Adding tooltip via ruby and rt element

### DISCUSSION THREAD

**Discussion by spurushottam13** [muxinc/media-chrome#932] [Proposals]
We can also use the  combination of `<ruby/>` and `<rt/>` elements, which allows the `<rt/>` element to be rendered on top of the content of `<ruby/>` element,
- we can by default using css hide the `<rt/>` and  toggle the visibility on hovering the `<ruby/>`
- Demo: https://demo-tooltip.tiiny.site/

**Comment by heff** [comment]
Nice demo! 👍 [Browser support for ruby](https://caniuse.com/?search=ruby) is looking a little sketchy so I'm not sure about relying on that as the main option. But it looks like we're going to be digging into tool tips in the next few weeks to get some first class support built into media chrome.

Do you have any specific requirements for tool tips that would be good for us to know about?
